float32 FACE_GAUCHE
float32 FACE_DROITE
float32 GAUCHE_HAUT
float32 GAUCHE_BAS
float32 ARRIERE_DROITE
float32 ARRIERE_GAUCHE
float32 DROITE_HAUT
float32 DROITE_BAS